pytorch_lightning/loggers/base.py

```diff
@@ -41,6 +42,13 @@ def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None):
         """
         pass
 
+    def _convert_params(self, params: Union[dict, Namespace]) -> dict:
+        assert isinstance(params, (dict, Namespace))
```

Comment:
I'm not sure this is a good idea. It goes against the whole idea of duck typing. Won't this throw an exception if I try to use e.g. an OrderedDict or defaultdict?

Reply:
Not sure, but typing is also not strict, right?
Ok I will remove it and extend Dict